frontend/Dockerfile.dev

```diff
@@ -1,15 +1,7 @@
-# Specify a base image
-FROM node:alpine 
-
-# Create working directory and copy the app before running npm install as the artifactory
-# credentials can be inside .npmrc
+FROM node:16
```

Comment:
Have you try node:16-alpine? I think alpine is preferable since it's smaller in size so the build time will be shorter.

Reply:
Yup, just tried and it works. Code changed accordingly.